sqitch.conf

```diff
@@ -5,10 +5,12 @@
 	# top_dir = .
 [engine "pg"]
 	target = sqitch_test
-	registry = sqitch
+	registry = sqitch_afyatek
```

Comment:
Why is this change necessary?

Reply:
I did this a while back when I was deploying to the stage schema on the production DB. The sqitch stuff on the production DB is in sqitch_afyatek, and not sqitch.